tutorials/plot_background_statistics.py

```diff
@@ -383,8 +382,8 @@ def plot_t_p(t, p, title, mcc, axes=None):
 # made, as the entire cluster as a whole is used to reject the null.
 # Moreover, because the test statistic concerns the full data, the null
 # hypothesis (and our rejection of it) refers to the structure of the full
-# data. For more information, see also the
-# `excellent FieldTrip cluster interpretation tutorial <ft_cluster>`_.
+# data. For more information, see also the excellent
```

Comment:
Is the "excellent" adequate style here? Isn't "comprehensive" or "authoritative" better?